backend/src/js/table_webcomponent.js

```
@@ -0,0 +1,412 @@
+                const zoomTable = zoomContent.querySelector('table');
+                if (window.$ && window.$.fn.DataTable && window.$.fn.DataTable.isDataTable(zoomTable)) {
+                    window.$(zoomTable).DataTable().destroy();
+                    zoomTable.dataset.initialized = '';
+                }
+            });
+
+            document.addEventListener('keydown', (event) => {
+                if (event.key === 'Escape' || event.key === 'Esc') {
+                    zoomOverlay.style.display = "none";
+                    const zoomTable = zoomContent.querySelector('table');
+                    if (window.$ && window.$.fn.DataTable && window.$.fn.DataTable.isDataTable(zoomTable)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)